tests/e2e/e2e_fluentbit_out_test.go

```diff
@@ -0,0 +1,98 @@
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	. "github.com/openservicemesh/osm/tests/framework"
+)
+
+var _ = OSMDescribe("Test deployment of Fluent Bit sidecar",
+	OSMDescribeInfo{
+		Tier:   2,
+		Bucket: 2,
+	},
+	func() {
+		Context("FluentbitOutput", func() {
+			It("Forwards correctly filtered logs to stdout", func() {
+				// Install OSM with Fluentbit
+				installOpts := Td.GetOSMInstallOpts()
+				installOpts.DeployFluentbit = true
+				installOpts.SetOverrides = []string{"OpenServiceMesh.fluentBit.logLevel=info"}
+				Expect(Td.InstallOSM(installOpts)).To(Succeed())
+
+				pods, err := Td.Client.CoreV1().Pods(Td.OsmNamespace).List(context.TODO(), metav1.ListOptions{
+					LabelSelector: labels.SelectorFromSet(map[string]string{"app": "osm-controller"}).String(),
+				})
+				Expect(err).NotTo(HaveOccurred())
+
+				// Query fluentbit-logger container logs to test if Fluent Bit filters are working
+				fluentBitContainerFound := false
+				for _, pod := range pods.Items {
+					// Wait until osm-controller has generated a log to check against
+					logLevel := "\"level\":\"info\""
+					err := waitForLogEmission(pod.Namespace, pod.Name, "osm-controller", logLevel)
```

Comment:
This wait might actually not be needed. Since the osm-controller will definitely log some info messages before it becomes ready, and `Td.InstallOSM` already waits for the pod to be ready, by the time you get here the pod should already have logged some messages. Not a big deal though to check anyway.

This assumption is unique to info messages though, so if the test is ever changed to look for different levels, this will need to be reevaluated.

Reply:
So I did see the test fail once because fluent bit took a bit longer to log to stdout than osm-controller. I think the poll gives it extra time to ensure that it won't be flakey. 

What you're saying is def a fair assumption, but I think it's also worth leaving in there in the case of changing log level. Ideally it would be useful to change this to test error level in the future. Since it is info rn, the wait should return true immediately based on what you're saying